src/bots/ama/constants.js

```diff
@@ -9,23 +9,59 @@ export const getStrings = (...options) => {
   options = options || [{}];
   return {
     createdEvent: `Following event has been created:\n${options[0]}`,
+    editEventExample: `Surround values with triple backticks and split by double newline. For example:\n${options[0]}`,
+    eventNotFound: `Event with id ${options[0]} not found`,
     futureEvents: `__**Future Events**__\n${options[0]}`,
     insufficientArgumentsAddEvent:
       'Need to supply date (yyyy-mm-dd) and title of event\n_Example_: 2020-01-01 start of the greatest year ever',
+    insufficientArgumentsEvent: 'Need to supply event ID',
     invalidDateAddEvent: 'Invalid date provided. Must be in format yyyy-mm-dd',
+    noBackticks: 'Must surround data with backticks (codeblock)',
     noEvents: 'No events yet :(',
     pastEvents: `__**Past Events**__\n${options[0]}`,
+    successfullyDeleted: `Event with id ${options[0]} successfully deleted`,
     upcomingEvent: `__**Upcoming Event**__\n${options[0]}\n`
   };
 };
 
+/**
+ * Possible edit fields
+ */
+export const possibleEditFields = dedent(
+  `++ama edit 5fd3f9a4ea601010fe5875ff
```

Comment:
will we have to resupply all the information every time we edit? If so, should probably improve in the future

Reply:
no just supply what you want to modify

Reply:
cool 👍